backend/pom.xml

```diff
@@ -62,6 +68,33 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
check whether all dependencies are needed. I mean for shared toolbox we can use other dependencies.